### PR TITLE
V2.0.0 rc3

### PR DESCRIPTION
**Pull request by hrvolapeter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,60 +4,54 @@
...
+**2** |**[`addons/`](addons/)** | **Complement** your landing zone with add-ons to run OCI with best practices. (e.g., [Network Hubs](/addons/oci-hub-models/readme.md))|
+| **3** |**[`workload-extensions/`](workload-extensions/)** | **Extend** your landing zone with pluggable workload extensions (e.g., [EBS](/workload-extensions/oci-lz-ext-ebs/readme.md), [OCVS](/workload-extensions/oci-lz-ext-ocvs/README.md), [...](/workload-extensions/readme.md)).|
 
 &nbsp; 
 
-## Where Do You Start
+As general guidance, it's recommended to start with the order of the table above. The first step is choosing the right **blueprint** shape and size (M, L, or XL). Complementary, review the network **add-ons** to update the design. After the Landing Zone is set, extend it with **workload extensions**. You can also use the [OCI-tailored Landing Zone approach](https://github.com/oracle-devrel/technology-engineering/tree/main/landing-zones/tailored_landing_zones) for further guidance on the creation of your landing zone.
```
Comment (hrvolapeter) [inline README.md:33]: maybe rather `Additionally, review the network` complementary doesn't feel right I guess @rphibbert  can help

File: addons/readme.md [hunk 2]
```diff
@@ -4,14 +4,14 @@
...
 
 &nbsp; 
 
-| ADD-ON  | DESCRIPTION | CONTENT |
-|---|---|---|
-| **[OCI Learn LZ](/addons/oci-learn-lz/readme.md)**| A Landing Zone **training** to learn how to design and run a Landing Zone without code. | Public |
-| **[OCI Network Hubs](/addons/oci-hub-models/readme.md)** | A well-documented set of **OCI Network Hub topologies** that can be applicable to any landing zone or OCI deployment. | Public | 
+| # |  Add-on   | Description | Status |
+|:--:|--|-|---|
+| 1 | <img src="oci-learn-lz/diagrams/oci_learn_lz_icon.svg" height="26" align="center"> &nbsp;   **[OCI Learn LZ](/addons/oci-learn-lz/readme.md)**| A Landing Zone **training** to learn how to design and run a Landing Zone without code. | [Available](/addons/oci-learn-lz/readme.md) |
```
Comment (hrvolapeter) [inline addons/readme.md:13]: how about `A Landing Zone **training** to learn how to design and run a Landing Zone with configuration files`

File: blueprints/one-oe/design/readme.md [hunk 3]
```diff
@@ -227,7 +227,7 @@ For more details on the network design refer to the [OCI Open LZ One-OE Drawio -
 This chapter will be added soon.
 
 &nbsp; 
-&nbsp; 
+&nbsp; v
```
Comment (hrvolapeter) [inline blueprints/one-oe/design/readme.md:230]: extra v

File: blueprints/one-oe/runtime/one-stack/oci_open_lz_one-oe_iam.auto.tfvars.json [hunk 4]
```diff
@@ -61,46 +61,46 @@
...
                     "CMP-UAT-KEY": {
-                        "name": "cmp-uat",
-                        "description": "one-oe-lz Prod Environment Compartment",
+                        "name": "cmp-lzp-uat",
+                        "description": "One-OE UAT Environment Compartment",
                         "children": {
                             "CMP-U-NETWORK-KEY": {
-                                "name": "cmp-u-network",
-                                "description": "one-oe-lz LZ Prod, UAT Environment, Common Network Compartment"                      
+                                "name": "cmp-lzp-uat-network",
```
Comment (hrvolapeter) [inline blueprints/one-oe/runtime/one-stack/oci_open_lz_one-oe_iam.auto.tfvars.json:68]: we use one letter for environment name, I think it'd be better to stick to `u`

File: blueprints/one-oe/runtime/one-stack/oci_open_lz_one-oe_iam.auto.tfvars.json [hunk 5]
```diff
@@ -61,46 +61,46 @@
...
                     "CMP-DEV-KEY": {
-                        "name": "cmp-dev",
-                        "description": "one-oe-lz Prod Environment Compartment",
+                        "name": "cmp-lzp-dev",
+                        "description": "One-OE Dev Environment Compartment",
                         "children": {
                             "CMP-D-NETWORK-KEY": {
-                                "name": "cmp-u-network",
-                                "description": "one-oe-lz LZ Prod, DEV Environment, Common Network Compartment"                      
+                                "name": "cmp-lzp-dev-network",
```
Comment (hrvolapeter) [inline blueprints/one-oe/runtime/one-stack/oci_open_lz_one-oe_iam.auto.tfvars.json:90]: this was an error and should have been `d` or make consistent with production which currently uses `p`

**Review by paalonso — APPROVED** [review]

**Pull request merged**